commands/host.vegman

```diff
@@ -224,6 +224,90 @@ function cmd_virtualmedia_umount {
   usb-ctrl eject "usb.ms.cli"
 }
 
+
+function config_default_set {
+  local yes=""
+  while [[ $# -gt 0 ]]; do
+    case "$1" in
+      -y | --yes) yes="y";;
+      *) abort_badarg "$1";;
+    esac
+    shift
+  done
```

Comment:
You don't need this anymore. You either pass `-y` in `$1` (that is, a non-empty `$1`), or you don't pass anything (empty `$1`). You only need this:

```
local yes="${1-}"

if [[ -z "${yes}" ]]; then
...
```

That was the whole idea behind determining the action in the main function that you don't have to repeat the code here and in cancel function.